examples/how_to/benchmark_with_hybrid_recordings.py

```diff
@@ -0,0 +1,258 @@
+static_recording, drifting_recording, sorting = sgen.generate_drifting_recording(
+    probe_name="Neuropixel-384",  # ,'Neuropixel-384',
+    seed=23,
+    duration=240,
+    num_units=100,
+    generate_displacement_vector_kwargs=generate_displacement_vector_kwargs,
+)
+
+# we sort the channels by depth, to match the  hybrid templates
+drifting_recording = spre.depth_order(drifting_recording)
+# -
+
+# To visualize the drift, we can estimate the motion and plot it:
+
+_, motion_info = spre.correct_motion(
+    drifting_recording, preset="nonrigid_fast_and_accurate", n_jobs=4, progress_bar=True, output_motion_info=True
+)
+
+
+ax = sw.plot_drift_map(
+    peaks=motion_info["peaks"],
+    peak_locations=motion_info["peak_locations"],
+    recording=drifting_recording,
+    cmap="Greys_r",
+)
+
+# ## Retrieve templates from database
+
+templates_info = sgen.fetch_templates_database_info()
+
+print(len(templates_info))
+
+templates_info.head()
+
+available_brain_areas = np.unique(templates_info.brain_area)
+print(f"Available brain areas: {available_brain_areas}")
+
+# let's perform a query: templates from brain region VISp5 and at the "top" of the probe
+target_area = ["VISa5", "VISa6a", "VISp5", "VISp6a", "VISrl6b"]
+minimum_depth = 1500
+templates_selected_info = templates_info.query(f"brain_area in {target_area} and depth_along_probe > {minimum_depth}")
+len(templates_selected_info)
+
+# We can now retrieve the selected templates as a `Templates` object
+#
+
+templates_selected = sgen.query_templates_from_database(templates_selected_info, verbose=True)
+print(templates_selected)
+
+# While we selected templates from a target aread and at certain depths, we can see that the template amplitudes are quite large. This will make spike sorting easy... we can further manipulate the `Templates` by rescaling, relocating, or further selections with the `sgen.scale_template_to_range`, `sgen.relocate_templates`, and `sgen.select_templates` functions.
+#
+# In our case, let's rescale the amplitudes between 50 and 150 $\mu$V and relocate them throughout the entire depth of the probe.
+
+min_amplitude = 50
+max_amplitude = 150
+templates_scaled = sgen.scale_template_to_range(
+    templates=templates_selected,
+    min_amplitude=min_amplitude,
+    max_amplitude=max_amplitude
+)
+
+min_displacement = 200
+max_displacement = 4000
+templates_relocated = sgen.relocate_templates(
+    templates=templates_scaled,
+    min_displacement=min_displacement,
+    max_displacement=max_displacement
+)
+
+# Let's plot the selected templates:
+
+sparsity_plot = si.compute_sparsity(templates_relocated)
+fig = plt.figure(figsize=(10, 10))
+w = sw.plot_unit_templates(templates_relocated, sparsity=sparsity_plot, ncols=4, figure=fig)
+w.figure.subplots_adjust(wspace=0.5, hspace=0.7)
+
+# ## Constructing hybrid recordings
+
+recording_hybrid_no_drift, sorting_hybrid = sgen.generate_hybrid_recording(
+    recording=drifting_recording, templates=templates_relocated, seed=2308
```

Comment:
should be the none drifting no ?

Reply:
no this is the drifting recording without accounting for drift. Good point though, maybe it needs a name change